AdyenComponents/Apple Pay/ApplePayConfiguration.swift

```diff
@@ -23,46 +24,55 @@ extension ApplePayComponent {
 
         /// A list of fields that you need for a billing contact in order to process the transaction.
         /// Ignored on iOS 10.*.
-        public let requiredBillingContactFields: Set<PKContactField>
+        public var requiredBillingContactFields: Set<PKContactField> = []
 
         /// A list of fields that you need for a shipping contact in order to process the transaction.
         /// Ignored on iOS 10.*.
-        public let requiredShippingContactFields: Set<PKContactField>
+        public var requiredShippingContactFields: Set<PKContactField> = []
 
         /// A pre-populated billing address.
-        public let billingContact: PKContact?
+        public var billingContact: PKContact?
 
         /// The flag to toggle onboarding.
         /// If true, allow the shopper to add cards to Apple Pay if non exists yet.
         /// If false, then Apple Pay is disabled if the shopper doesn't have supported cards on Apple Pay wallet.
-        public let allowOnboarding: Bool
+        /// Default is false.
+        public var allowOnboarding: Bool = false
+
+        // Shipping methods supported by the merchant.
+        public var shippingContact: PKContact?
+
+        // Indicates the display mode for the shipping (e.g, "Pick Up", "Ship To", "Deliver To"). Localized.
+        // The default is PKShippingTypeShipping
+        public var shippingType: PKShippingType = .shipping
+
+        // Indicates the editing mode for the shipping contact. The default is true.
+        public let shippingContactEditing: Bool = true
```

Comment:
This name and description isn't clear to me of what the flag is supposed to do.

Reply:
Renamed to `allowShippingContactEditing`